xml/vt_installation.xml

```diff
@@ -258,4 +258,75 @@
    />.
   </para>
  </sect1>
+ <sect1 xml:id="sec-vt-installation-nested-vms">
+  <title>Enable Support for Nested Virtualization in &kvm;</title>
+  <important>
+   <title>Technology Preview</title>
+   <para>
+    &kvm;'s nested virtualization is still a technology preview.  It is
+    provided for testing purposes and is not supported.
+   </para>
+  </important>
+  <para>
+   Nested guests are &kvm; guests run in a &kvm; guest. When describing nested
+   guests, we will use the following virtualization layers:
+  </para>
+  <variablelist>
+   <varlistentry>
+    <term>L0</term>
+    <listitem>
+     <para>
+      A bare metal host running &kvm;.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>L1</term>
+    <listitem>
+     <para>
+      A virtual machine running on L0. Because it can run another &kvm;, it is
+      called a <emphasis>guest hypervisor</emphasis>.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>L2</term>
+    <listitem>
+     <para>
+      A virtual machine running on L1. It is called a <emphasis>nested
+       guest</emphasis>.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+  <para>
+   Nested virtualization is useful mainly in cloud environments as you can
```

Comment:
Running your own hypervisor in a VM in the cloud is definitely one use-case, but dev/testing is another. Also I'd suggest dropping 'favorite' from this sentence and just go with "your hypervisor of choice".